cores/migrations/20240708182646_AddCategoryMarkdown.up.sql

```diff
@@ -0,0 +1,15 @@
+-- Drop the existing primary key constraint
+ALTER TABLE Category DROP CONSTRAINT IF EXISTS category_pkey;
+
+-- Ensure Category table has category_id column with a default UUID value
+ALTER TABLE Category ADD COLUMN IF NOT EXISTS category_id UUID DEFAULT gen_random_uuid() NOT NULL;
+
+-- Set category_id as the new primary key
+ALTER TABLE Category ADD CONSTRAINT category_pkey PRIMARY KEY (category_id);
```

Comment:
I'm a _bit_ worried about this... 🤔 

Do you know/expect the Category table in Prod to be empty? If so then this is probably fine! But if not I don't think we can change out the Primary Key like this 🤔